unlock-app/src/components/interface/buttons/lock/index.js

```diff
@@ -7,6 +7,7 @@ import Upload from './Upload'
 import Etherscan from './Etherscan'
 import Preview from './Preview'
 import Copy from './Copy'
+import Save from './Save'
```

Comment:
Wait... that was not included before? sorry :/ 
I can certainly pull that into its own PR so that it's less confusing. LMK.

Reply:
Would you like me to make it a separate commit or a separate PR to add a save button?
Because we also need to choose a decent SVG and I'm not sure if you drew all these or found them someplace.

Reply:
Wait actually you got me confused here. I think you do not need a save SVG button?

Here's what I think would work:
<img width="907" alt="screenshot 2018-12-16 at 21 52 47" src="https://user-images.githubusercontent.com/17735/50064033-fa64e700-017c-11e9-8f6e-e533d510054a.png">
(please ignore the email stuff)

Basically when the user clicks on the pencil button, we toggle all fields back to form inputs (same UI/layout as when creating the lock), and the button bar is replaced with the grey area with both "Update" and "Cancel". Does it make sense?

Reply:
Yes this looks good